todo_app/templates/index.html

```diff
@@ -25,19 +25,22 @@ <h1 class="display-4"><b>To-Do App</b></h1>
       </style>
```

Comment:
Not really necessary to change now, but usually it's cleaner to include css styles in a separate file